### ISSUE TITLE
VMAgent sends all EC2 filters in GetAZMap function, it fails DescribeAvailabilityZones request and __meta_ec2_availability_zone_id label is empty

### ISSUE THREAD

**Issue by lgalaktionov** [VictoriaMetrics/VictoriaMetrics#1626]
**Describe the bug**
VMAgent sends all EC2 filters in [GetAZMap](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/948fb638f56e9ae70512da3025f9e55d4275b126/lib/promscrape/discovery/ec2/instance.go#L136
) function, but AWS DescribeAvailabilityZones API permits only [limited list of filters](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeAvailabilityZones.html).

**To Reproduce**
Create scrape config with additional filters for EC2 instances:
```yaml
  - job_name: asd
    ec2_sd_configs:
      - region: eu-west-2
        port: 9201
        filters:
        - name: 'tag:project'
          values: ['asd']
        - name: 'tag:role'
          values: ['asd-server’]
      - region: eu-north-1
        port: 9201
        filters:
        - name: 'tag:project'
          values: ['asd']
        - name: 'tag:role'
          values: ['asd-server’]
    relabel_configs:
      - source_labels: [__meta_ec2_tag_Name]
        target_label: type
      - source_labels: [__meta_ec2_public_dns_name]
        target_label: instance
      - source_labels: [__meta_ec2_availability_zone]
        regex: (.*[0-9]+).*
        replacement: ${1}
        target_label: region
```

**Expected behavior**
No additional tags, except permitted by AWS is sent in DescribeAvailabilityZones request.

**Logs**
```
2021-09-16T19:27:35.125Z warn VictoriaMetrics/lib/promscrape/discovery/ec2/instance.go:147 couldn't load availability zones map, so __meta_ec2_availability_zone_id label isn't set: cannot obtain availability zones: unexpected status code for "https://ec2.eu-west-2.amazonaws.com/?Action=DescribeAvailabilityZones&Filter.1.Name=tag%3Aproject&Filter.1.Value.1=asd&Filter.2.Name=tag%3Arole&Filter.2.Value.1=asd-server&Version=2013-10-15"; got 400; want 200; response body: "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<Response><Errors><Error><Code>InvalidParameterValue</Code><Message>The filter 'tag:project' is invalid</Message></Error></Errors><RequestID>c18c193a-4db2-4aaa-936e-eb7338e5614d</RequestID></Response>”
```

**Screenshots**
-

**Version**
```
/ # /vmagent-prod --version
vmagent-20210901-141901-tags-v1.65.0-0-gf529058d3
```

**Used command-line flags**
```
      "-memory.allowedBytes",
      "1GiB",
      "-promscrape.config",
      "/config/vmagent.yml",
      "-remoteWrite.basicAuth.password",
      "PASS",
      "-remoteWrite.basicAuth.username",
      "USER",
      "-remoteWrite.tmpDataPath",
      "/data",
      "-remoteWrite.url",
      "https://victoriaURL"
```

**Comment by valyala** [comment]
The issue must be fixed in the commit 0e09fdb8b09a4290e246c4a92d985a2d27e22e5e . This commit will be included in the next release.

**Comment by valyala** [comment]
The bugfix has been included in [VictoriaMetrics v1.66.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing the issue as fixed.
